bundles/org.openhab.binding.robonect/README.md

```diff
@@ -66,7 +66,9 @@ Thing robonect:mower:automower "Mower" @ "Garden" [ host="192.168.2.1", pollInte
 | `last-error-date`         | DateTime  | The date and time of the last error happened                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
 | `health-temperature`         | Number    | The temperature of the mower (just available for robonect firmware >= 1.0)                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | `health-humidity`         | Number    | The humidity of the mower (just available for robonect firmware >= 1.0)                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-
+| `blades-quality`         | Number    | The quality of the blades                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `blades-days`         | Number    | The number of days since the blades have been changed                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Defined as `Number:Time` in thing-types.xml:
```suggestion
| `blades-days`         | Number:Time | The number of days since the blades have been changed                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```